src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1212,10 +1214,11 @@ static bool addTriggeredSubscriptions_withCache
   //
   // Now, take the 'patterned subscriptions' from the Subscription Cache and add more TriggeredSubscription to subs
   //
+#if SUB_CACHE_ON
   std::vector<Subscription*> subVec;
 
   subCache->semTake();
-  //subCache->lookup(tenant, servicePath, entityId, entityType, attr, &subVec);
+  subCache->lookup(tenant, servicePath, entityId, entityType, attr, &subVec);
 
```

Comment:
Yes, and the whole 'block' is removed by the

    #if SUB_CACHE_ON
    #endif